tests/integration/test_integration_gnmi.py

```diff
@@ -36,3 +39,40 @@ async def test_integration_get(service):
     )
     assert isinstance(response, gnmi.proto.GetResponse)
     validate_default_interfaces_get(response)
+
+
+async def test_integration_update_set_string(service):
+    new_password = str(uuid.uuid4())
+    path = gnmi.proto.Path(
+        elem=[
+            gnmi.proto.PathElem(name="system"),
+            gnmi.proto.PathElem(name="aaa"),
+            gnmi.proto.PathElem(name="authentication"),
+            gnmi.proto.PathElem(name="admin-user"),
+            gnmi.proto.PathElem(name="config"),
+            gnmi.proto.PathElem(name="admin-password"),
+        ],
+    )
+    update = gnmi.proto.Update(
+        path=path, val=gnmi.proto.TypedValue(string_val=new_password)
+    )
+    response = await service.set(update=[update],)
```

Comment:
there is a redundant comma

Reply:
Not really redundant, it is expected since black reformats something like this:
```py
method(
    key=value,
)
```

To something like:
```py
method(key=value,)
```

Keeping the comma is just cleaner practice.

Reply:
I see. Always good to learn something new.